docs/VERSION_UPDATE_CHECKLIST.md

```
@@ -64,46 +64,148 @@ calibre-debug -g
 
 ### 第四步：更新文档
 
+#### 4.1 创建 CHANGELOG 文档（必须）
+
+**🔴 重要：每次版本更新都必须创建新的 CHANGELOG 文档**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)